pkg/controller/node/status_test.go

```diff
@@ -159,7 +159,7 @@ func TestGetUpdatedMachines(t *testing.T) {
 
```

Comment:
**thought:** This test suite seems like the most appropriate place to write a couple of unit test cases for your change. I can help with that, if needed.